sympy/polys/domains/pythonintegerring.py

```diff
@@ -71,20 +75,28 @@ def from_RealField(K1, a, K0):
 
     def gcdex(self, a, b):
         """Compute extended GCD of ``a`` and ``b``. """
+        a = self.from_sympy(a)
```

Comment:
None of these conversions should be added. This should only be used for `PythonInteger` arguments. Otherwise an error should result to show that something is wrong.